<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualLEDConfigDlg</class>
 <widget class="QDialog" name="VirtualLEDConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LED配置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>字体大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSizeCmbBox">
         <item>
          <property name="text">
           <string>自适应</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自定义</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="fontSizeSlider">
         <property name="minimum">
          <number>48</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fontSizeLab">
         <property name="text">
          <string>48</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>字间距</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontGapCmbBox">
         <item>
          <property name="text">
           <string>自适应</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10x</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typefaceCmbBox">
         <item>
          <property name="text">
           <string>宋体</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>黑体</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>楷体</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arial</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>透明</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transparencyCmbBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>是</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>否</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>滚动模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ScrollModeCmbBox">
         <item>
          <property name="text">
           <string>静止</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>从右到左</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>从左到右</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>从上到下</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>从下到上</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>滚动速度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="scrollRateSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ScrollRateLab">
         <property name="text">
          <string>1倍速</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>字体颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fontColorBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>背景颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backgroundColorBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>字体对齐方式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="horizontalAlignCmbBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>左对齐</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>居中对齐</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>右对齐</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="verticalAlignCmbBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>上对齐</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>居中对齐</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>下对齐</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>文字内容</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="contentLineEdt"/>
       </item>
      </layout>
     </item>
     <item row="12" column="2">
      <widget class="QComboBox" name="enableCmbBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VirtualLEDConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VirtualLEDConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
